src/momento/requests/collection_ttl.py

```diff
@@ -82,19 +84,20 @@ def refresh_ttl_if_provided(ttl: Optional[timedelta] = None) -> CollectionTtl:
         return CollectionTtl(ttl=ttl, refresh_ttl=ttl is not None)
 
     def with_refresh_ttl_on_updates(self) -> CollectionTtl:
-        """Specifies that the TTL for the collection should be refreshed when
-        the collection is modified.  (This is the default behavior.)
+        """Specifies the TTL for the collection be refreshed when the collection is modified.
+
+        NB: This is the default behavior.
 
         Returns:
             CollectionTtl
         """
         return CollectionTtl(ttl=self.ttl, refresh_ttl=True)
 
     def with_no_refresh_ttl_on_updates(self) -> CollectionTtl:
-        """Specifies that the TTL for the collection should not be refreshed
-        when the collection is modified.  Use this if you want to ensure
-        that your collection expires at the originally specified time, even
-        if you make modifications to the value of the collection.
+        """Specifies the TTL for the collection should not be refreshed when the collection is modified.
+
+        Use this if ou want to ensure that your collection expires at the originally specified time,
+        even if you make modifications to the value of the collection.
```

Comment:
I feel like limiting this to only when the values are modified will bite the user when we inevitably decide to apply refresh on delete or something. YMMV

```suggestion
        Use this to ensure your collection expires at the originally specified time,
        even if you make modifications to the collection.
```